app/views/podcasts/index.html.erb

```diff
@@ -1,14 +1,40 @@
+<% content_for :title, "My Podcasts" %>
+
 <p style="color: green"><%= notice %></p>
+<div class="row mt-4 mb-4">
+  <div class="dashboard header">
+    <div class="col d-flex align-items-center ">
+      <h1 class="text-black fw-bold me-4 mb-0">My Podcasts</h1>
```

Comment:
Maybe a design question, but "My" may not be true - this is podcasts they can access, not some kind of implied ownership

Reply:
I deferred to the design, but I can change it if you like